### PR TITLE
Feature/add av1 video codec

### PR DESCRIPTION
**Pull request by alodijutgla**
Creation of the feature branch _add_AV1_VideoCodec_ in order to add the AV1 codec configurations to the encoding configurations.

### CODE REVIEW

File: src/main/java/com/bitmovin/api/encoding/codecConfigurations/AV1VideoConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,168 @@
+package com.bitmovin.api.encoding.codecConfigurations;
+
+import com.bitmovin.api.encoding.codecConfigurations.enums.*;
```
Comment (msmole) [inline src/main/java/com/bitmovin/api/encoding/codecConfigurations/AV1VideoConfiguration.java]: Do not use * imports.

**Review by ChristianFeldmann — CHANGES_REQUESTED** [review]
Don't we have to add AV1 also to the bitmovin-java/src/main/java/com/bitmovin/api/encoding/codecConfigurations/enums/ConfigType.java file? Or is this part of something else?

File: src/main/java/com/bitmovin/api/encoding/codecConfigurations/enums/AV1KeyPlacementMode.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+ * Al·lodi Jutgla Serrat [allodi.serrat@bitmovin.com]
+ * on 16.08.18.
+ */
+
+public enum AV1KeyPlacementMode
+{
+    AUTO,
+    FIXED,
+    DISABLED
+
```
Comment (msmole) [inline src/main/java/com/bitmovin/api/encoding/codecConfigurations/enums/AV1KeyPlacementMode.java]: remove empty line

File: src/main/java/com/bitmovin/api/encoding/codecConfigurations/AV1VideoConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+/**
```
Comment (msmole) [inline src/main/java/com/bitmovin/api/encoding/codecConfigurations/AV1VideoConfiguration.java]: Remove all header comments.

File: src/main/java/com/bitmovin/api/encoding/codecConfigurations/enums/AV1AdaptiveQuantMode.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
+package com.bitmovin.api.encoding.codecConfigurations.enums;
+
+/**
```
Comment (msmole) [inline src/main/java/com/bitmovin/api/encoding/codecConfigurations/enums/AV1AdaptiveQuantMode.java]: Remove

File: src/main/java/com/bitmovin/api/encoding/codecConfigurations/enums/AV1KeyPlacementMode.java [hunk 5]
```diff
@@ -0,0 +1,15 @@
+package com.bitmovin.api.encoding.codecConfigurations.enums;
+
+/**
```
Comment (msmole) [inline src/main/java/com/bitmovin/api/encoding/codecConfigurations/enums/AV1KeyPlacementMode.java]: Remove

**Review by msmole — CHANGES_REQUESTED** [review]

**Review by ChristianFeldmann — APPROVED** [review]

File: src/main/java/com/bitmovin/api/encoding/codecConfigurations/AV1VideoConfiguration.java [hunk 6]
```diff
@@ -0,0 +1,159 @@
+package com.bitmovin.api.encoding.codecConfigurations;
+
+
+public class AV1VideoConfiguration extends VideoConfiguration {
```
Comment (msmole) [inline src/main/java/com/bitmovin/api/encoding/codecConfigurations/AV1VideoConfiguration.java]: `{` in next line.

File: src/main/java/com/bitmovin/api/encoding/codecConfigurations/AV1VideoConfiguration.java [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+    private Integer sharpness;
+    private Boolean isFrameboostEnabled;
+    private Boolean noiseSensitivity;
+    private Integer minGfInterval;
+    private Integer maxGfInterval;
+    private Integer numTileGroups;
+    private Integer mtuSize;
+
+
+    public Integer getLagInFrames() { return this.lagInFrames; }
```
Comment (msmole) [inline src/main/java/com/bitmovin/api/encoding/codecConfigurations/AV1VideoConfiguration.java]: Please write getters and setters not as one-liner.
`{` always on a new line. After that the statement, and then the `}` on a new line again.

**Review by msmole — CHANGES_REQUESTED** [review]

**Review by msmole — APPROVED** [review]

**Pull request merged**